lib/util.js

```diff
@@ -2,7 +2,7 @@ const Log = require('./logging')
 const { spawnSync } = require('child_process')
 
 const getNPMConfig = (path) => {
-  const key = path.join('_').replace('-', '_')
+  const key = path.join('_')
```

Comment:
I don't believe this is the correct fix, see https://github.com/brave/brave-core/commit/5cbf5fc88a921ee02773a3652c2e0e1ce5fc62ea

Reply:
Sorry @bridiver, could you add more information on this? Are you saying that brave-browser should be refactored to requiring `package.json` instead of accessing variables using `process.env`?

Reply:
I'm saying that just doing `path.join('_')` is not correct. It should do `path.join('-').replace(/_/g, '-')` like the brave-core version. You can see in that change that it was the same as this method. Ideally they'd both use a common method, but this stuff will go away soon anyway.

Reply:
Sure I agree that both brave-browser and brave-core should use the same type of method to look up the repo url, but that won't work right out of the box. Change the string replacement to produce `projects-brave-core-repository-url` instead of `projects_brave-core_repository_url` and make code changes in brave-browser util.js to support that new format?

Reply:
`npm run init` works fine for me without this change, but it fails with it. This is also working fine on our CI machines where we do fresh checkouts on a regular basis

Reply:
@bridiver Thank you for the hand holding, it's much appreciated! No longer able to reproduce on my end, I'd be happy with closing if no one is able to reproduce.